src/js/options.js

```diff
@@ -1,8 +1,3 @@
-function recommendExercise() {
-  // data massage and do recommendations
-  console.log('TODO')
-}
-
```

Comment:
Remove this for now. Next PR we will show the youtube link when the timer is up.